src/algebra/monoid_algebra.lean

```diff
@@ -62,6 +63,9 @@ namespace monoid_algebra
 
 variables {k G}
 
+instance {R : Type*} [semiring R] [semiring k] [module R k] : module R (monoid_algebra k G) :=
+finsupp.module G k
```

Comment:
I think this made more sense where it was, and reduced the chance of repeating work.

Reply:
Fair enough. I'll have to move the new stuff below the `derived_instances` section because it needs the `distrib_mul_action` instance (in fact `module` is overkill). I'll try that and see how things look.